python_agent/test_custom_env.py

```
@@ -0,0 +1,154 @@
+        self.mock_zmq_client.execute_action.reset_mock()
+
+        # Prime the next observation after action (bones gone)
+        obs_after_pickup = self._get_default_raw_obs() 
+        obs_after_pickup["nearby_ground_items"] = [] # Bones are gone
+        # This will be returned for the _get_obs call at the end of the step method
+        self.mock_zmq_client.get_observation.return_value = obs_after_pickup
+        
+        _, reward, _, _, info = self.env.step(5) # Action 5 = pickup_bone
+        
+        self.mock_zmq_client.execute_action.assert_called_once_with(
+            "interact_ground_item", {"item_id": BONE_ITEM_ID}
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)